base_images/fedora_base-setup.sh

```diff
@@ -35,15 +34,10 @@ if ! ((CONTAINER)); then
     fi
 fi
 
-# Due to https://bugzilla.redhat.com/show_bug.cgi?id=1907030
-# updates cannot be installed or even looked at during this stage.
-# Pawn the problem off to the cache-image stage where more memory
-# is available and debugging is also easier.  Try to save some more
-# memory by pre-populating repo metadata prior to any transactions.
-$SUDO dnf makecache $XARGS
-# Updates disable, see comment above
-# $SUDO dnf -y update $XARGS
-$SUDO dnf -y install $XARGS "${PKGS[@]}"
+$SUDO dnf makecache
+$SUDO dnf -y update
+$SUDO dnf -y install "${PKGS[@]}"
+$SUDO dnf -y update
```

Comment:
This too looks like a leftover. If it is intentional, could you add a comment explaining what it's intended to do?

Reply:
This one is on purpose, I'll add a comment.